cmd/tracee-ebpf/main.go

```diff
@@ -30,6 +32,12 @@ var buildPolicy string
 var version string
 
 func main() {
+	err := config.LoadConfig(config.DefaultConfigLocation())
+
+	if err != nil {
+		log.Println("No configuration file found in environment variable or default location, flags will be set from cli.")
```

Comment:
let's use fmt.Printf as in the rest of the file

Reply:
1. Alright
2. I think all the error values here would be arbitrary strings (depends on the implementation of the std packages used to parse), and i'm not sure what we could do with them besides notifying the user differently. At first I logged a fatal here but I thought this shouldn't stop the user, would you say it should?

Reply:
logic changed a bit, new logs reflect that